tests/conformance/scripts/generate-inputs.ts

```diff
@@ -144,10 +144,11 @@ function generateFile(features: Set<TestFeatures>, records: TestDataRecord[]) {
   const statisticsStart = BigInt(builder.length);
   if (features.has(TestFeatures.UseStatistics)) {
     builder.writeStatistics({
-      attachmentCount,
-      chunkCount,
       messageCount,
       channelCount,
+      attachmentCount,
+      metadataCount: 0,
```

Comment:
it's not clear to me why this gets a default but not attachment count

Reply:
though I guess the code does support it. so I should update this and in the future we can add test files with metadata easily